apps/server/src/routes/ping.ts

```diff
@@ -1,5 +1,7 @@
 import { Router } from 'express';
 
+import * as pingController from '../controllers/pingController.js';
```

Comment:
not sure why this `* as var` seems smelly to me. for some reason I feel we should export all the desired items as properties of a default object export in that other file, as if that somehow makes it more explicit for someone looking into what's available there (like a book's "index").

Reply:
fixed this as suggested.